crates/tools/coding-agent-tools/CLAUDE.md

```
@@ -25,6 +25,11 @@ just fmt          # Format code
 - Pagination state in struct; CLI creates fresh instance (no pagination), MCP reuses Arc-wrapped instance
 - McpFormatter for token-efficient text output
 
+## Search Ignore Policy
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)